less/project/theme-extras.less

```diff
@@ -157,41 +157,87 @@
 // --------------------------------------------------
 .block-color-mixin(@color, @color-inverted) {
 
-.block-color.@{color} {
-  background-color: @@color;
-  color: @@color-inverted;
+  .block-color.@{color} {
+    background-color: @@color;
+    color: @@color-inverted;
 
-  .block,
-  .component {
-    &__title,
-    &__body a,
-    &__instruction a {
-      color: @@color-inverted;
+    .block,
+    .component {
+      &__title,
+      &__body a,
+      &__instruction a {
+        color: @@color-inverted;
+      }
     }
-  }
 
-  .pagelevelprogress {
-    &__indicator {
-      border-color: @@color-inverted;
-    }
-    &__indicator-inner {
-      background-color: @@color;
+    .pagelevelprogress {
+      &__indicator {
+        border-color: @@color-inverted;
+      }
+      &__indicator-inner {
+        background-color: @@color;
+      }
+      &__indicator-bar {
+        background-color: @@color-inverted;
+      }
     }
-    &__indicator-bar {
-      background-color: @@color-inverted;
+
+    .narrative {
+      &__progress {
+        background-color: fade(@@color-inverted, 50%);
+      }
+
+      &__progress.is-selected {
+        background-color: @@color-inverted;
+      }
     }
   }
 
-  .narrative {
-    &__progress {
-      background-color: fade(@@color-inverted, 50%);
+}
+
+// --------------------------------------------------
+// Component
+// --------------------------------------------------
+
+// Component header bg color mixin
```

Comment:
The addition of component header style and various style/spacing tweaks could be moved out into a separate PR as it isn't column related?

Reply:
Agreed with splitting into a separate PR.

It was included for the time being so 1) it didn't get lost / forgotten about and 2) helpful for development